### PR TITLE
Tweaks some vendors to better use Contraband and premium items.

### PR DESCRIPTION
**Pull request by Pacmandevil**

### CODE REVIEW

**Comment by OneOneThreeEight** [comment]
this just adds fuzzy cuffs...

cmon man

File: code/game/machinery/vending_types.dm [hunk 1]
```diff
@@ -505,12 +512,12 @@
 		/obj/item/weapon/material/knife = 3,
 		/obj/item/weapon/reagent_containers/food/drinks/drinkingglass = 8,
 		/obj/item/clothing/suit/chef/classic = 2,
+		/obj/item/weapon/material/kitchen/rollingpin = 2,
```
Comment (Pacmandevil) [inline code/game/machinery/vending_types.dm]: I'm not sure why a rolling pin was in the contraband section in the first place. unless I missed the reasoning behind it.

**Comment by Pacmandevil** [comment]
travis please

**Comment by Lohikar** [comment]
Would be nice if the various cuff messages were adjusted to actually specify what type of cuffs the perp was wearing too. Most seem to just say "cuffs".

**Comment by Arrow768** [comment]
I think thats going to conflict with #3795 when merged.

**Comment by Pacmandevil** [comment]
Doesn't look like it did, @Arrow768

**Comment by Alberyk** [comment]
No handscuffs.

**Comment by skull132** [comment]
Please don't bring recycled meme-garbage from reddit here.

**Comment by Pacmandevil** [comment]
any humor's been removed. this should be good for review.

**Comment by Arrow768** [comment]
The controversial item has been removed.
Just needs a second review now.

**Pull request merged**